src/rpcrequest-payload.ts

```diff
@@ -34,19 +35,10 @@ class RpcRequestPayload {
   }
 
   getStream() {
-    const self = this;
-
-    return new Readable({
-      read() {
-        self.getData((buf) => {
-          this.push(buf);
-          this.push(null);
-        });
-      }
-    });
+    return Readable.from(this.generateData());
```

Comment:
```suggestion
    return Readable.from(this.generateData(), { objectMode: false });
```

According to https://nodejs.org/dist/latest-v12.x/docs/api/stream.html#stream_stream_readable_from_iterable_options, a `Readable` stream created by `Readable.from` has by default `objectMode: true`. But as we're working with `Buffer` data exclusively, changing it to `objectMode: false` is preferred.